setup.py

```diff
@@ -33,12 +33,10 @@
         "typing_extensions",
         "packaging",
         "deprecated",
+        "dataclasses; python_version < "3.7",
```

Comment:
This doesn't look right. Maybe something like this:
```suggestion
        "dataclasses; python_version < '3.7'",
```
Our continuous integration tools are also complaining.
Maybe you can test this on your local machine?

Reply:
Apologies! I will correct and run the test locally

Reply:
This looks much better. Thanks!